examples/online_judge_programming/submit.py

```
@@ -0,0 +1,603 @@
+            print(f"\r{status_text}...", end="")
+        else:
+            print("\rTest cases: ", end="")
+
+        if status_id == _COMPILE_ERROR_STATUS:
+            print(f"\r{color(status_text, _RED_COLOR)}", end="")
+            try:
+                root = fragment_fromstring(status["feedback_html"], create_parent=True)
+                error = root.find(".//pre").text
+                print(color(":", _RED_COLOR))
+                print(error, end="")
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)